.travis.yml

```diff
@@ -44,8 +44,8 @@ before_script:
   - proj | head -n1
   - lua -v
 script:
-  ./autogen.sh && ./configure && make -j2 check TESTS=''
+  ./autogen.sh && ./configure && make -j2 check TESTS='' CXXFLAGS="-Werror -pedantic -Wall -std=c++11"
```

Comment:
CXXFLAGS can't be set to have -pedantic since it cause some configure checks to fail, so all of the flags need to be overridden here.